src/cryojax/simulator/_detector.py

```diff
@@ -43,7 +48,12 @@ def __init__(self):
         self.fraction_detected_electrons = jnp.asarray(1.0)
 
     @override
-    def __call__(self, frequency_grid: ImageCoords) -> Real_:
+    def __call__(
+        self,
+        frequency_grid_maybe_in_angstroms: ImageCoords,
```

Comment:
maybe??? It's better than nothing, I suppose.

angstroms_or_wavenumbers???

Reply:
the reason why I have this is a messy detail I don't quite have resolved in the AbstractFourierOperator class... will document this